src/nn/oe.rs

```diff
@@ -76,6 +93,17 @@ extern "C" {
     #[link_name = "\u{1}_ZN2nn2oe18GetDesiredLanguageEv"]
     pub fn GetDesiredLanguage() -> root::nn::settings::LanguageCode;
 }
+
+pub fn get_desired_language() -> String {
+    let mut result = "";
+
+    let lang_code = unsafe { GetDesiredLanguage() };
+
+    result = core::str::from_utf8(&lang_code.code).unwrap();
```

Comment:
I suspect this will be a null-terminated string, so you’ll probably want to use CString/CStr beforehand.